templates/summary.html

```diff
@@ -3,7 +3,7 @@
 
 {% import 'macros/helpers.html' as helpers %}
 
-{% set summary_page_title = _("Summary") %}
+{% set summary_page_title = _("Submit census") %}
```

Comment:
This should come from the context.
```suggestion
{% set summary_page_title = content.title %}
```